const-oid/src/lib.rs

```diff
@@ -29,6 +29,12 @@ pub use crate::{
 use crate::encoder::Encoder;
 use core::{fmt, str::FromStr};
 
+/// Identifies a type with a given OID.
+pub trait Identified {
```

Comment:
I can't think of a better name myself, but I think this trait would be clearer if it had `Oid` somewhere in the name.

Reply:
Object Identifier Identified? :)

Reply:
`Identified` is an extremely vague name for a trait

Reply:
@tarcieri How about the name `AssociatedOid` which I just pushed?

Reply:
Sounds good, thanks